### ISSUE TITLE
/lib/libc.so.6: version `GLIBC_2.18' not found

### ISSUE THREAD

**Issue by izrie1** [bmisiak/samp-precise-timers#9]
[20:24:20]  Loading plugin: samp-precise-timers.so
[20:24:20]   Failed (/lib/libc.so.6: version `GLIBC_2.18' not found (required by plugins/samp-precise-timers.so))

 I have this error. I use CentOS 7.5 for my server. Need help

**Comment by MacLove13** [comment]
UP

**Comment by bmisiak** [comment]
For CentOS, we’d probably benefit from compiling for musl. Want to give it a go? It’s really simple. Install rustup as mentioned in the README and replace `gnu` in the instruction with `musl`:
```
rustup target add i686-unknown-linux-musl
cargo build --target=i686-unknown-linux-musl --release
```

**Comment by bmisiak** [comment]
The issue stems from the fact that the plugin is built on a relatively new OS and linked against GLIBC 2.18, but CentOS 7 ships with 2.17 at best.

Immediate solution: try upgrading to CentOS 8.2, which ships with GLIBC 2.28.

When I have some free time I'll set up a build machine which will build against an older glibc.

**Comment by bmisiak** [comment]
@izrie1 @MacLove13 the issue is fixed in v2.0.2. Thanks!
